### PR TITLE
layout_home & layout_profile

### PR DESCRIPTION
**Pull request by DuyNguyen05**
login

### CODE REVIEW

**Comment by DuyNguyen05** [comment]
he

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class SessionsController < ApplicationController
+
+  def new
```
Comment (tunm1180) [inline app/controllers/sessions_controller.rb]: sai convention

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class UsersController < ApplicationController
+
+  def show
```
Comment (tunm1180) [inline app/controllers/users_controller.rb]: same above

**Comment by DuyNguyen05** [comment]
dashboard

**Review by tunm1180 — APPROVED** [review]

**Pull request merged**